pkg/controller/advancedcronjob/advancedcronjob_controller_test.go

```diff
@@ -0,0 +1,193 @@
+/*
+Copyright 2019 The Kruise Authors.
```

Comment:
2019->2020

Reply:
Done in commit https://github.com/openkruise/kruise/pull/468/commits/aa5bc2c7c7d02e38c348cbbb7134aa9d8b76c80c